extension/api.js

```diff
@@ -38,17 +38,17 @@ HideFilesOnGitHub.storage = {
 };
 
 // Inlined partial `escape-goat` package
-window.escape = input => input
+const escapeHTML = input => input
 	.replace(/&/g, '&amp;')
 	.replace(/"/g, '&quot;')
 	.replace(/'/g, '&#39;')
 	.replace(/</g, '&lt;')
 	.replace(/>/g, '&gt;');
 
-window.escapeTag = (input, ...parts) => {
+window.escapeTag = function (input, ...parts) {
```

Comment:
Huh? This got reverted?

Reply:
Ah yeah, I opened an _edit_ page before merging your other suggestion.